frontend/src/components/Header/Header.tsx

```diff
@@ -44,32 +49,70 @@ export function Header(): JSX.Element {
 
   const [open, setOpen] = useState(false);
   const [listItems, setListItems] = useState([
-    { link: "/", text: t("homepage") }
+    { link: "/", text: t("homepage") },
   ]);
 
-  useEffect(() => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLButtonElement>(
+    null,
+  );
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  React.useEffect(() => {
     const baseItems = [{ link: "/", text: t("homepage") }];
     const additionalItems = [
       { link: "/projects", text: t("myProjects") },
-      { link: "/courses", text: t("myCourses") }
+      { link: "/courses", text: t("myCourses") },
     ];
-    if (isLoggedIn()) {
+    if (me.role !== "UNKNOWN") {
```

Comment:
No, this must check if person is logged in, don't check the role. Or check the role inside the `isLoggedIn` function

Reply:
Not solved